theia-cpp-docker/Dockerfile

```diff
@@ -112,9 +112,9 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
     ln -s /usr/bin/clangd-$LLVM /usr/bin/clangd
 
 # Install latest stable CMake
-ARG CMAKE_VERSION=3.17.3
+ARG CMAKE_VERSION=3.18.1
 
-RUN wget "https://cmake.org/files/v3.17/cmake-$CMAKE_VERSION-Linux-x86_64.sh" && \
+RUN wget "https://cmake.org/files/v3.18/cmake-$CMAKE_VERSION-Linux-x86_64.sh" && \
```

Comment:
From the [CMake download page](https://cmake.org/download/), I found this URL that could be used instead so we do not need to update the CMakeversion in two places for each image: 

https://github.com/Kitware/CMake/releases/download/v3.18.1/cmake-3.18.1-Linux-x86_64.sh

e.g.:
https://github.com/Kitware/CMake/releases/download/v$CMAKE_VERSION/cmake-$CMAKE_VERSION-Linux-x86_64.sh